_reports/RJ722.md

```diff
@@ -0,0 +1,388 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+date: 2018-08-12
+report_year: 2018
+title: Rahul Jha
+student: Rahul Jha
+organisation: coala
+organisation_link : https://coala.io
+project: Improve Whitelisting Infrastructure for Vulture
+project_link: https://summerofcode.withgoogle.com/projects/#5620743946108928
+tarball: https://drive.google.com/file/d/12HgBb2Wdjl6Zzm7mDqCt13ke8VHj7_Bb/view
+mentors: >
+ [Jendrik Seipp](https://github.com/jendrikseipp), [Abdeali Kothari](https://github.com/AbdealiJK), [Florian Bruhin](https://github.com/The-Compiler)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/28
```

Comment:
indentation must pass yamllint

Reply:
actually, all my scripts for editing would go havoc if I change it right now. 😅 Would indent it properly once I've addressed all the review comments.

Reply:
still not done.

Reply:
Done! sorry for the delay, was asleep.